README.md

```diff
@@ -29,7 +29,15 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+require 'pry'
+class User
+  def offer_rose (person)
+    puts "Would you take this rose, #{person} in exchange for giving an old beggar woman shelter from the bitter cold?"
+  end
+  young_prince = User.new("Young Prince")
```

Comment:
Close, we can do a little less here. No need for the class definition. Can just define `offer_rose` as you have and call with the string `"Young Prince"`
``` ruby
def offer_rose (person)
  puts "Would you take this rose, #{person} in exchange for giving an old beggar woman shelter from the bitter cold?"
end

offer_rose("Young Prince")
```